src/request/request-builder.ts

```diff
@@ -0,0 +1,142 @@
+import testRunTracker from '../api/test-run-tracker';
```

Comment:
We don't need an additional library for sending request.
Use the [DestinationRequest](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/request-pipeline/destination-request/index.ts) from hammerhead.